app/content/views/registration.py

```diff
@@ -94,3 +98,13 @@ def _admin_unregister(self, registration):
             {"detail": "Brukeren har blitt meldt av arrangement"},
             status=status.HTTP_200_OK,
         )
+
+    @action(detail=True, methods=["get"], url_path="strikes")
+    def get_registration_strikes(self, request, *args, **kwargs):
+        registration = self.get_object()
+        strikes = Strike.objects.filter(
+            user=registration.user, event=registration.event
```

Comment:
Det gir ikke så mye verdi å kun se prikkene for dette arrangementet, NoK trenger å se alle aktive prikker for brukeren.

Reply:
Tenker å lage en Manager så vi enkelt kan få tak i alle aktive strikes og slippe det under og litt annet

Reply:
@olros  hva gjør vi med denne